std/typecons.d

```diff
@@ -568,6 +594,13 @@ template Tuple(Specs...)
          *   $(LI For each value `v1` on the left-hand side and each value 
          *        `v2` on the right-hand side, the expression `v1 == v2` is 
          *        true.))
+         *
+         * Params:
+         *     rhs = The `Tuple` to compare against. It must meeting the criteria
+         *           for comparison between `Tuple`s.
+         *
+         * Returns:
+         *     true if both `Tuples` are equal, otherwise false.
```

Comment:
But this is specifically how Tuple comparison is implemented; if each of its members are equal.